homeassistant/components/tuya/strings.json

```diff
@@ -0,0 +1,26 @@
+{
+  "config": {
+    "flow_title": "Tuya configuration",
+    "step": {
+      "user": {
+        "title": "Tuya",
+        "description": "Enter your Tuya credential.",
+        "data": {
+          "country_code": "Your account country code (e.g., 1 for USA or 86 for China)",
+          "password": "Your password to log in to Tuya",
+          "platform": "The app where your account register",
+          "username": "Your username to log in to Tuya"
+        }
+      }
+    },
+    "abort": {
+      "already_in_progress": "Tuya configuration is already in progress.",
+      "auth_failed": "Configured Tuya credential are incorrect.",
+      "conn_error": "Connection to Tuya failed.",
+      "single_instance_allowed": "Only a single configuration of Tuya is allowed."
+    },
+    "error": {
+      "auth_failed": "Provided credential are incorrect."
+    }
+  }
+}
```

Comment:
You can use common strings where applicable, see https://github.com/home-assistant/core/pull/34523 for examples on how.

Reply:
I used common string, can you check to be sure that I correctly applied?

Reply:
LGTM